modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/BookmarksFragment.kt

```diff
@@ -44,21 +81,198 @@ class BookmarksFragment : BaseFragment() {
+    }
+}
+
+@Composable
+private fun Content(
+    bookmarks: List<Bookmark>,
+    backgroundColor: Color,
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = backgroundColor)
+    ) {
+        item {
+            val title = stringResource(
+                id = if (bookmarks.size > 1) {
+                    LR.string.bookmarks_plural
+                } else {
+                    LR.string.bookmarks_singular
+                },
+                bookmarks.size
+            )
+            HeaderItem(title)
+        }
+        items(bookmarks) { bookmark ->
+            BookmarkItem(
+                bookmark = bookmark,
+                textColor = backgroundColor
+            )
+        }
+    }
+}
+
+@Composable
+private fun HeaderItem(title: String) {
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 16.dp, end = 4.dp, top = 4.dp, bottom = 4.dp)
+    ) {
+        TextH40(
+            text = title,
+            color = MaterialTheme.theme.colors.playerContrast02,
+        )
+        IconButton(
+            onClick = { /* TODO */ },
+        ) {
+            Icon(
+                painter = painterResource(IR.drawable.ic_more_vert_black_24dp),
+                contentDescription = stringResource(LR.string.more_options),
+                tint = MaterialTheme.theme.colors.playerContrast01,
+            )
+        }
+    }
+}
+
+@Composable
+private fun BookmarkItem(
+    bookmark: Bookmark,
+    textColor: Color,
+) {
+    Column(
+        modifier = Modifier
+            .clickable { /* TODO */ }
+    ) {
+        Spacer(
+            modifier = Modifier
+                .height(1.dp)
+                .fillMaxWidth()
+                .background(color = MaterialTheme.theme.colors.playerContrast05)
+        )
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier
-                .fillMaxSize()
-                .background(Color(it.podcastHeader.backgroundColor))
+                .fillMaxWidth()
+                .padding(16.dp)
         ) {
-            TextH30(
-                text = "Placeholder Text",
-                color = MaterialTheme.theme.colors.playerContrast01
+            val createdAtText by remember {
+                mutableStateOf(
+                    bookmark.createdAt.toLocalizedFormatPattern(
+                        pattern = "MMM d, YYYY 'at' h:mm a"
+                    )
+                )
+            }
+            Column(
+                Modifier.weight(1f, fill = true)
+            ) {
+                TextH40(
+                    text = bookmark.title,
+                    color = MaterialTheme.theme.colors.playerContrast01,
+                    maxLines = 2,
+                )
```

Comment:
It handled long titles well! 
<img width="584" alt="Screenshot 2023-07-05 at 10 50 23 am" src="https://github.com/Automattic/pocket-casts-android/assets/308331/a6e9b03e-0971-4d52-8901-afe9d3c9a998">